angular/projects/spark-angular/src/lib/components/sprk-toggle/sprk-toggle.component.ts

```diff
@@ -56,26 +58,69 @@ export class SprkToggleComponent implements AfterViewInit {
    */
   @Input()
   additionalClasses: string;
+  // TODO: Remove `title` in issue #1305
   /**
+   * Deprecated - Use `triggerText` instead.
    * The value supplied will be
-   * rendered as the main Toggle link text.
+   * rendered as the main Toggle button text.
    */
   @Input()
   title: string;
   /**
-   * The value supplied will be assigned as a
-   * CSS class on the icon used in the Toggle.
-   * This is intended for overrides.
+   * The value supplied will be
+   * rendered as the main Toggle button text.
+   */
+  @Input()
+  triggerText: string;
+  // TODO: Remove `iconClass` in issue #1305
+  /**
+   * Deprecated - Use `iconAdditionalClasses` instead.
+   * Expects a space separated string
+   * of classes to be added to the
+   * icon.
    */
   @Input()
   iconClass: string;
   /**
-   * The value supplied will be assigned as a CSS class
-   * on the clickable title text used in the Toggle.
-   * This is intended for overrides.
+   * Expects a space separated string
+   * of classes to be added to the
+   * icon.
+   */
+  @Input()
+  iconAdditionalClasses: string;
+  /**
+   * Determines which icon is rendered.
+   * Expects the value to match the exact name
+   * of the icon found in the docs
+   * (i.e. `chevron-down`, instead of `chevron down`).
+   */
+  @Input()
+  toggleIconName = 'chevron-down-circle';
+  /**
+   * Expects a space separated string
+   * of classes to be added to the
+   * content.
+   */
+  @Input()
+  contentAdditionalClasses: string;
+  // TODO: Remove `titleFontClass` in future issue #1305.
+  /**
+   * Deprecated - Use `titleAdditionalClasses` instead.
+   * Expects a space separated string
+   * of classes to be added to the
+   * trigger text.
    */
   @Input()
   titleFontClass = 'sprk-b-TypeBodyThree';
+  // TODO: Move the default value from titleFontClass to
+  // titleAdditionalClasses in future issue #1305.
+  /**
+   * Expects a space separated string
+   * of classes to be added to the
+   * trigger text.
+   */
+  @Input()
+  titleAdditionalClasses: string;
```

Comment:
hey @guern1kn , In going through this, a light bulb went off haha. If we change title to heading we have to change the other names too like titleAdditionalClasses